docs/static/releasenotes.asciidoc

```diff
@@ -24,6 +25,108 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-6-0]]
+=== Logstash 7.6.0 Release Notes
+
+* Feature: Introduce deprecation logger for internal classes and plugins. https://github.com/elastic/logstash/pull/11260[#11260] and https://github.com/elastic/logstash/pull/11486[#11486]
+
+** The Deprecation logger is a unified way for Logstash components to log deprecation notices into a separate file,
+ located by default at `log/logstash-deprecation.log`. This file gives users a single location to see if they are using features that may stop working after a major upgrade. "Deprecations are noted in the log/logstash-deprecation.log file.
```

Comment:
The file location info is in twice.  If you choose to keep the second instance, the `"` should be deleted.